### ISSUE TITLE
Nunit.consoleruuner can't run .net core tests

### ISSUE THREAD

**Issue by santhosh-James** [nunit/nunit-console#656]
Is there any update on .net console runner in .net core?

**Comment by mikkelbu** [comment]
Not much as far as I know. I guess that the issue to watch is #478, but that is still blocked by #266 - "Planning: replacing remoting with custom protocol". There is a PR for #266, but that has been idle for some time (it is a major change and I guess that the best solution is not obvious).

**Comment by ChrisMaddock** [comment]
@mikkelbu's got it about right.

In the meantime, you can use `dotnet test` - here's the docs: https://github.com/nunit/docs/wiki/.NET-Core-and-.NET-Standard

I'll close this issue as a duplicate of #478 - please follow that for any further updates.
